tests/implement/tests/vector_view.rs

```diff
@@ -0,0 +1,141 @@
+use std::vec;
+use test_implement::*;
+use windows::*;
+use Windows::Foundation::Collections::*;
+
+#[implement(
+    Windows::Foundation::Collections::IVectorView<T>,
```

Comment:
`IVectorView<T>` requires `IIterable<T>`. Soon the `implement` macro will pick this up automatically, but for now you'll just have to list that here as well.

Reply:
Added the type, and we need to implement First, which requires an implementation of an IIterator. Used the same implementation as in `into_impl.rs`. Is that expected?

Reply:
Yes, that's the basic outline - `IIterable<T>` must be implemented on the same object while `IIterator<T>` must be implemented on a seperate object.